windows/bleachbit.nsi

```diff
@@ -17,163 +17,731 @@
 ;  You should have received a copy of the GNU General Public License
 ;  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+;  @app BleachBit NSIS Installer Script
+;  @url https://nsis.sourceforge.io/Main_Page
+;  @os Windows
+;  @scriptversion v2.3.1040
+;  @scriptdate 2019-04-03
+;  @scriptby Andrew Ziem (2009-05-14 - 2019-01-21) & Tobias B. Besemer (2019-03-31 - 2019-04-03)
+;  @tested ok v2.3.1039, Windows 7
+;  @testeddate 2019-04-03
+;  @testedby https://github.com/Tobias-B-Besemer
+;  @note 
```

Comment:
I do not see a benefit to this section compared to what's already in Git. For example, Git already documents the authors and the dates of their changes. A Git commit message is a good place to add more, like how you tested it. I don't want to maintain this information.

Reply:
I think you don't need to maintain this information! ^^
When I'm done with the installer, I think there's nothing more to do in next time! ^^
Maybe I'm oldish, but I like those headers and even played with the idea to add a EOF like in many NSIS-examples I found...
I think people will download the source/the files, have it on there computer, work with it or spread it further... A header with version info is therefor IMHO always a good thing!
If you talk just about the test-comments: I copied the header from the cleaners... I can remove the test-stuff again, I don't need it!